querybook/webapp/components/Search/SearchOverview.tsx

```diff
@@ -112,6 +112,18 @@ export const SearchOverview: React.FC<ISearchOverviewProps> = ({
     const results = resultByPage[currentPage] || [];
     const isLoading = !!searchRequest;
 
+    // Log search results
+    useEffect(() => {
+        if (!isLoading && !!searchString.length && !!results.length) {
```

Comment:
can you use searchString.length > 0 and results.length >0, seems easier to understand than `!!`